### ISSUE TITLE
Warnings in console when using textfield

### ISSUE THREAD

**Issue by adrianicv** [n4kz/react-native-material-textfield#63]
Using the component fires these warnings into the console:

```
Warning: Unknown props `animationDuration`, `active`, `focused`, `errored` on <div> tag. Remove these props from the element. For details, see https://fb.me/react-unknown-prop
    in div (created by Text)
    in Text (created by AnimatedComponent)
    in AnimatedComponent (at index.js:121)
    in div (created by View)
```
The problem may be that the component is passing all props to the text native component.

**Comment by adrianicv** [comment]
#64 will fix this

**Comment by n4kz** [comment]
Thanks for reporting! Merged #70 and released in `0.10.3`.
